usmqe/web/utils.py

```diff
@@ -0,0 +1,37 @@
+"""
+Some usefull methods and classes for common work with web
+"""
+
+from selenium.common import exceptions as selenium_ex
+from webstr.selenium.driver import Driver
+
+
+def refresh_driver(testcase, relogin=False):
+    """
+    Reload page or log again and go back to the volumes tab
+    if reload of the page fails because there is no page present
+    it logs again anyway
+
+    Atributes:
+        testcase - instance of UITestCase
```

Comment:
right, it is. I thought about that and decided against, however I could/will change it